New York Times article clone

sjcco opened the pull request:
New York Times article clone
In this proyect I:
Used positioning and floating elements such as:

- flexbox
- grid
- float

to recreate an article page from NYT

Mayor languages used:

- HTML
- CSS

sjcco committed:
Create linters.yml

set up html&css linter

sjcco committed:
assets folder

added assets folder (images and icons)

sjcco committed:
Create index.html

added  markup skeleton

sjcco committed:
Create styles.css

added style file

sjcco committed:
Mark up

added finished Markup of the document

sjcco committed:
added assets

resized title image

sjcco committed:
css

Added flexbox and css structure

sjcco committed:
Update styles.css

lintern error correction

sjcco committed:
update css html

lintern error correction

sjcco committed:
Update styles.css

linter error correction

sjcco committed:
Update styles.css

fixed lintern error

sjcco committed:
Update styles.css

fixed linter error

sjcco committed:
update css html

fixed linter error

sjcco committed:
Update index.html

fixed linter error

sjcco committed:
Update index.html

fixed linter error

sjcco committed:
fixed

fixed editors pick and top bar

sjcco committed:
Update styles.css

fixed linter error

sjcco committed:
update

fixed linter error

umairarshadbutt reviewed (changes requested):
# Status: REQUIRED CHANGES :warning: 

**Hi @slavkojos  AND @ashraftanin20 **
The project is nice :clap:  However there is a problem with your Pull Request.
Kindly read about PR ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) and fix your project by adding PR description and updating the readme file
I am going to mark this project as invalid so when you fix the issue just submit another code review request.
Please Read particularly [Article](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
Use Readme file [template](https://github.com/microverseinc/readme-template)

##

 **If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

sjcco committed:
update

updated README

sjcco committed:
Update README.md

updated README

sjcco committed:
update

fixed linter error

sjcco committed:
Update styles.css

linter error

SpaYco reviewed (changes requested):
# Changes Requested 

Hello! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

* [x] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) Talk about what you've made in this branch/project, and what kind of technologies you used
* [x] fix the live demo link in your README.md, you can use http://raw.githack.com/
* [x] The NY Times logo should be a link ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/positioning_and_floating_requirements.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/))
* [x] the **The Theory of inflation** section should be one image with texts aligned on top of it (you can use background-image, and display grid for this) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))
* [x] Elements should be aligned with a combination of all three techniques ([float](https://css-tricks.com/all-about-floats/), [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) and [grid](https://css-tricks.com/snippets/css/complete-guide-grid/)). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_pathwright.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538403/)) and ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/positioning_and_floating_elements/flex_float_grid_requirement_odin.png) from [Odin](https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements#introduction)) you don't have any use of grid at the moment.
* [x] you have some overflow on the right (extra space), please fix that.
* [x] you have a lot of anchor tags with an empty `href`, please make sure it has something, the most used thing to fill an empty `href` is `#`

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

sjcco committed:
update

fixed logo now takes you the NYT homepage
fixed The theory of inflation is now one image
fixed use of grid technique
fixed used "#" in href tags

sjcco committed:
Update styles.css

working on right overflow

sjcco committed:
Update styles.css

fixed linter error

sjcco committed:
Update index.html

fixed linter error

sjcco committed:
Update README.md

fixed link to live demo

sjcco committed:
Update styles.css

fixed right overflow

Georjane reviewed (approved):
## Status: PR Approved ✔️ ✔️
### Hi Team 🙌
Congratulations on successfully completing this project. You have made the changes required by the previous TSE. It's a thumbs up for me.👍
You can go ahead and merge this PR.

Goodluck on your next project and Happy Coding 💻
